### PR TITLE
new fictores for collect and show datas

### PR DESCRIPTION
**Pull request by adrissonGalvao**

### CODE REVIEW

File: tests/sensor_test.go [hunk 1]
```diff
@@ -20,9 +20,55 @@ func TestVerifyIntegrityUrlSensor(t *testing.T) {
...
+			Convey("verifying that the URLs were created correctly", func() {
+				Convey("Url gererated correctly", func() {
+					So("/"+sensors[0].Url, ShouldEqual, resultTest[0])
+				})
+			})
+		})
+	})
+}
+
+/*func TestGeneratingUrlSensors(t *testing.T) {
```
Comment (edvmorango) [inline tests/sensor_test.go]: @adrissonGalvao please remove this comments.

File: tests/sensor_test.go [hunk 2]
```diff
@@ -6,11 +6,11 @@ import (
 	mock "home-collect/tests/mocks"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/smartystreets/goconvey/convey"
 	"gopkg.in/mgo.v2/bson"
 )
 
-func TestVerifyIntegrityUrlSensor(t *testing.T) {
+/*func TestVerifyIntegrityUrlSensor(t *testing.T) {
```
Comment (edvmorango) [inline tests/sensor_test.go]: @adrissonGalvao please remove this comments.

**Review by edvmorango — CHANGES_REQUESTED** [review]

File: tests/sensor_test.go (hunk 2 above)
Comment (adrissonGalvao) [reply tests/sensor_test.go]: fixed it

File: tests/sensor_test.go (hunk 1 above)
Comment (adrissonGalvao) [reply tests/sensor_test.go]: fixed it

**Review by edvmorango — APPROVED** [review]

**Pull request merged**